cloud/pkg/cloudhub/servers/httpserver/server.go

```diff
@@ -86,8 +86,9 @@ func EncodeCertPEM(cert *x509.Certificate) []byte {
 
 // edgeCoreClientCert will verify the certificate of EdgeCore or token then create EdgeCoreCert and return it
 func edgeCoreClientCert(request *restful.Request, response *restful.Response) {
+	nodeName := request.Request.Header.Get("NodeName")
```

Comment:
Please use `request.Request.Header.Get(constants.NodeName)` and replase the repeatedly defined code with the nodeName

Reply:
> Please use `request.Request.Header.Get(constants.NodeName)` and replase the repeatedly defined code with the nodeName

fixed.

Reply:
You can replace codes `request.Request.Header.Get(constants.NodeName)` with `nodeName` at L93 and L106

Reply:
> You can replace codes `request.Request.Header.Get(constants.NodeName)` with `nodeName` at L93 and L106

Done. And I moved the `constants.NodeName` to `types.NodeNameKey` in `common/types/http.go`, since I think it is more appropriate to place the request const in http.go same as the` AuthorizationKey` const.